components/webfield/SeniorAreaChairConsole.js

```diff
@@ -259,22 +261,26 @@ const SeniorAreaChairConsole = ({ appContext }) => {
             }
           }),
           secondaries: areaChairGroup.members.flatMap((member) => {
-
-            if (secondaryAreaChairName && member.endsWith(`/${secondaryAreaChairName}`)) {
-              const secondaryAreaChairGroup = secondaryAreaChairGroups.find(
-                (p) => p.noteNumber === areaChairGroup.noteNumber
-              )
-              if (secondaryAreaChairGroup) {
-                return secondaryAreaChairGroup.members.map((secondaryMember) => ({
-                    areaChairProfileId: secondaryAnonAreaChairGroups[areaChairGroup.noteNumber]?.[secondaryMember]
-                    ?? secondaryMember,
-                    anonymizedGroup: secondaryMember,
-                    anonymousId: getIndentifierFromGroup(secondaryMember, secondaryAnonAreaChairName),
-                  }
-                ))
-              }
+            if (!secondaryAreaChairName || !member.endsWith(`/${secondaryAreaChairName}`)) {
+              return []
             }
-            return []
+
+            const acGroupNoteNum = areaChairGroup.noteNumber
+            const secondaryAreaChairGroup = secondaryAreaChairGroups.find(
+              (p) => p.noteNumber === acGroupNoteNum
+            )
+            if (secondaryAreaChairGroup) return []
```

Comment:
next time let me review the changes. we don't have tests to verify the refactor works

Reply:
Anyway you should test all the changes that you make.